docs/job-scheduling.md

```diff
@@ -101,6 +101,11 @@ then set `yarn.nodemanager.aux-services.spark_shuffle.class` to
 `spark.shuffle.service.*` [configurations](configuration.html).
 4. Restart all `NodeManager`s in your cluster.
 
+If you wish to use the external shuffle service in a Mesos deployment you need to launch the
+service on each Mesos slave that might run Spark jobs. A launcher script is provided in the `sbin/`
+directory: `bin/start-shuffle-service.sh`. You can later stop the service by running
+`sbin/stop-shuffle-service.sh`. Additional configuration options can be passed in `SPARK_SHUFFLE_OPTS`.
+
```

Comment:
This seems to be the wrong patch to put this. As of the changes here dynamic allocation is not yet supported in Mesos yet, so it would make more sense to put this paragraph in your other patch. Otherwise if this patch goes in first then the docs are a little misleading, especially since in L72 we explicitly say that DA on Mesos is not supported yet.